draco/core/containers.py

```diff
@@ -1064,6 +1067,73 @@ def phi(self):
         return self.index_map["phi"]
 
 
+class HEALPixBeam(ContainerBase):
+    """Generic container for representing the 2-d beam in spherical
+    coordinates on a HEALPix grid.
+    """
+
+    _axes = ("freq", "pol", "input", "pix")
+
+    _dataset_spec = {
+        "beam": {
+            "axes": ["freq", "pol", "input", "pix"],
+            "dtype": [("Et", np.complex64), ("Ep", np.complex64)],
+            "initialise": True,
+            "distributed": True,
+            "distributed_axis": "freq",
+        },
+        "weight": {
+            "axes": ["freq", "pol", "input", "pix"],
+            "dtype": [("Et", np.float32), ("Ep", np.float32)],
+            "initialise": False,
+            "distributed": True,
+            "distributed_axis": "freq",
+        },
+    }
+
+    def __init__(self, coords="unknown", ordering="unknown", *args, **kwargs):
+
+        super(HEALPixBeam, self).__init__(*args, **kwargs)
+        self.attrs["coords"] = coords
+        self.attrs["ordering"] = ordering
```

Comment:
You should document (in the class docstring) what `coords` and `ordering` are for (listing possible values if that's reasonable).